src/test/java/com/teamsfinder/userreadservice/user/service/UserServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package com.teamsfinder.userreadservice.user.service;
+
+import com.teamsfinder.userreadservice.user.UnitBaseClass;
+import com.teamsfinder.userreadservice.user.dto.UserResponseDto;
+import com.teamsfinder.userreadservice.user.model.AccountType;
+import com.teamsfinder.userreadservice.user.model.User;
+import com.teamsfinder.userreadservice.user.repository.UserRepository;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class UserServiceTest extends UnitBaseClass {
+
+    private static final String USER_KEYCLOAK_ID = "KEYCLOAK_ID";
+    private static final String USER_GITHUB = "GITHUB";
+    private static final String USER_PICTURE = "PICTURE";
+
+    @Mock
+    private UserRepository userRepository;
+
+    @InjectMocks
+    private UserService underTest;
+
+    private User testUser = User.builder()
+            .id(1L)
+            .keyCloakId(USER_KEYCLOAK_ID)
+            .accountType(AccountType.USER)
+            .githubProfileUrl(USER_GITHUB)
+            .profilePictureUrl(USER_PICTURE)
+            .blocked(false)
+            .tags(new ArrayList<>())
+            .build();
+
+    @Test
+    void getAllUsers() {
+        //given
+        Mockito.when(userRepository.findAll()).thenReturn(List.of(testUser));
```

Comment:
Może przenieś to Mockito do góry do importów masz wtedy po prostu np. when(...) wydaje mi się ze lepiej ze cos takiego czyta